src/pythae/models/beta_tc_vae/beta_tc_vae_model.py

```diff
@@ -79,7 +79,7 @@ def forward(self, inputs: BaseDataset, **kwargs):
         )
 
         output = ModelOutput(
-            reconstruction_loss=recon_loss,
```

Comment:
Update this [test](https://github.com/ravih18/pythae/blob/a1c8cce2be8e94ecb7283e7f65e568581b82df14/tests/test_BetaTCVAE.py#L295-L297) in `test_BetaTCVAE` with `recon_loss` to avoid failure